src/attached_probe.cpp

```diff
@@ -333,15 +334,20 @@ void AttachedProbe::load_prog()
   if (strrchr(name, ':') != NULL)
     namep = strrchr(name, ':') + 1;
 
+  snprintf(log_buf, sizeof(log_buf), "could not determine kernel version");
   for (int attempt=0; attempt<3; attempt++)
   {
+    auto version_of_kernel = kernel_version(attempt);
```

Comment:
`version_of_kernel` just sounds a bit strange to me - maybe just `version` or some other shorter name